server.md

```diff
@@ -1,11 +1,34 @@
 
+$ cat ~/.sbt/1.0/server/0845deda85cb41abdb9f/token.json
 
-### ExecCommand
+### initialize
 
 ```json
-{ "type": "ExecCommand", "commandLine": "compile" }
+{ "jsonrpc": "2.0", "id": 1, "method": "initialize", "params": { "initializationOptions": { "token": "************" } } }
 ```
 
+### sbt/exec
+
+```json
+{ "jsonrpc": "2.0", "id": 1, "method": "sbt/exec", "params": { "commandLine": "compile" } }
+```
+
+### sbt/setting
+
+```json
+{ "jsonrpc": "2.0", "id": 1, "method": "sbt/setting", "params": { "setting": "root/name" } }
```

Comment:
sbt/query?

Reply:
I thought about keeping the name query in there, but the Microsoft style naming for query is using just the nouns it seems. See https://github.com/Microsoft/language-server-protocol/blob/master/protocol.md#workspace_symbol